distributed/ddp-tutorial-series/slurm/sbatch_run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#SBATCH --job-name=multinode-example
+#SBATCH --nodes=4
+#SBATCH --ntasks=4
+#SBATCH --gpus-per-task=1
+#SBATCH --cpus-per-task=4
+
+nodes=( $( scontrol show hostnames $SLURM_JOB_NODELIST ) )
+nodes_array=($nodes)
+head_node=${nodes_array[0]}
+head_node_ip=$(srun --nodes=1 --ntasks=1 -w "$head_node" hostname --ip-address)
+
+echo Node IP: $head_node_ip
+export LOGLEVEL=INFO
+
+srun torchrun \
+--nnodes 4 \
+--nproc_per_node 1 \
+--rdzv_id $RANDOM \
```

Comment:
did we confirm this works by running the script? Not 100% sure if $RANDOM will be the same across nodes, but we'd need that to be the case right?

Reply:
yes it works, confirmed on video :D my understanding is `srun` runs the same command on all nodes (and not computing $RANDOM at each node separately)